omod/src/main/webapp/resources/htmlforms/072-IntegratedMaternityRegister.xml

```diff
@@ -257,27 +406,27 @@
             <p>
               <label>Gravida</label>
               <br/>

```

Comment:
Why the consistent <br/> tag in between the label and the obs. Why not use the labelText attribute of the obs tag https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module+HTML+Reference#HTMLFormEntryModuleHTMLReference-<obs>